src/App.jsx

```diff
@@ -11,9 +11,10 @@ function App() {
     <>
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index path="/Home" element={<Home />} />
-          <Route path="/Calculator" element={<Calculator />} />
-          <Route path="/Quote" element={<Quote />} />
+          <Route index element={<Home />} />
+          <Route path="Home" element={<Home />} />
+          <Route path="Calculator" element={<Calculator />} />
+          <Route path="Quote" element={<Quote />} />
         </Route>
```

Comment:
[OPTIONAL]
- [ ] Kindly ensure your components have a descriptive name for a better understanding of your code base quickly for easy maintenance. Why not name `NavBar` instead of `Layout`? Even if the project works correctly, your codebase needs to be improved more efficiently. since the navigation bar is supposed to be present permanently on top of other components, I would suggest you implement your `App` components this way:

```jsx
function App() {
  return (
    <>
       < Layout />
      <Routes>
          <Route index element={<Home />} />
          <Route path="Calculator" element={<Calculator />} />
          <Route path="Quote" element={<Quote />} />
        </Route>
      </Routes>
    </>
  );
}
```
Instead of ;
```jsx
function App() {
  return (
    <>
      <Routes>
        <Route path="/" element={<Layout />}>
          <Route index element={<Home />} />
          <Route path="Home" element={<Home />} />
          <Route path="Calculator" element={<Calculator />} />
          <Route path="Quote" element={<Quote />} />
        </Route>

      </Routes>
    </>

  );
}
```